content/docs/introduction/monitoring-page.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Monitoring dashboard
+enableTableOfContents: true
+---
+
+The **Monitoring** dashboard in the Neon console provides several graphs for monitoring system and database metrics. You can access the **Monitoring** dashboard from the sidebar in the Neon Console. Observable metrics include:
+
+- [RAM](#ram)
+- [CPU](#cpu)
+- [Connections count](#connections-count)
+- [Buffer cache hit rate](#buffer-cache-hit-rate)
+- [Database size](#database-size)
+- [Deadlocks](#deadlocks)
+- [Rows](#rows)
+
+Your Neon plan defines the range of data you can view.
+
+| Neon Plan | Data Access  |
+|-----------|--------------|
+| [Free Tier](/docs/introduction/plans#free-tier) | Last hour     |
+| [Launch](/docs/introduction/plans#launch)    | Last day (24 hours)       |
+| [Scale](/docs/introduction/plans#scale)     | Last 7 days      |
```

Comment:
Current limits:

* Free: 24 hours
* Launch and Scale: 168 hours (7 days)

In the future we plane to increase Scale's limit to 14 days.

[Slack](https://neondb.slack.com/archives/C061CC0Q82J/p1711528036250199?thread_ts=1711526664.074259&cid=C061CC0Q82J)

Reply:
fixed